catroid/src/main/res/values/strings.xml

```diff
@@ -2051,8 +2051,9 @@ needs read and write access to it. You can always change permissions through you
     <string name="cast_screen_pause_text">Paused</string>
     <!-- -->
 
-    <string name="deletion_alert_title">Warning!</string>
-    <string name="deletion_alert_text">Deleting a variable that is still in use is dangerous. Are you sure?</string>
+    <string name="deletion_alert_title">Delete this variable?</string>
+    <string name="deletion_alert_text">Deleting a variable that is still in use could cause errors.
```

Comment:
```suggestion
    <string name="deletion_alert_title">Delete?</string>
    <string name="deletion_alert_text">Deleting a variable or list that is still in use can cause errors.
```
Reason: This text is shown also when more than one variables and/or lists were selected, so plurals would have to be considered, as well as lists mentioned, see below. Also, the proposed shorter title line is completely understandable from the context, i.e., from the option that the user selected in the overflow menu.
![image](https://user-images.githubusercontent.com/1747888/98777582-554afb80-23f1-11eb-9d44-c3bf6afd1ae7.png)

Note: There would be a way to differentiate between singulars and plurals in texts shown in Android, but that involves more code, and I also think that it is not necessary in this case as the phrase is understood equally well. In this case it would also be quite complicated, e.g., if one variable and two lists were selected etc. Also, in other languages it is not necessary, for instance in Japanese there is no difference between singular and plural. So, IMHO it is not be worth the effort to differentiate here between singular and plural.